pandas/tests/io/formats/style/test_matplotlib.py

```diff
@@ -10,8 +11,6 @@
 pytest.importorskip("matplotlib")
 pytest.importorskip("jinja2")
 
-import matplotlib as mpl
```

Comment:
Looks like this move is not safe as is - it was purposefully guarded by the line above `pytest.importorskip("matplotlib")`

Reply:
Since only in one place, easiest solution might just be to disable the check for this line and with a small comment explaining why